includes/wc-webhook-functions.php

```diff
@@ -202,3 +206,42 @@ function wc_get_webhook_rest_api_versions() {
 		'wp_api_v3',
 	);
 }
+
+/**
+ * Add retries on woo webhook fails (from woocommerce_webhook_deliver action)
+ * see \WC_Webhook::deliver() for the arguments.
+ *
+ * @param array $http_args HTTP Request arguments (e.g. method, timeout, redirection, etc.)
+ * @param WP_Error|HTTP_Response $response
+ * @param float $duration The length of time for the request to complete
+ * @param int $arg The CPT id
+ * @param int $webhook_id
+ */
+function wc_requeue_failed_webhook_delivery(
```

Comment:
What would you think of performing the requeue operation in `WC_Webhook::failed_delivery()`?

Reply:
Yes, I agree, let me know if my approach (calling the delivery hook) is ok.  I wanted to get access to all those variables 😬 .